### PR TITLE
Sonar profile en 1964

### PR DESCRIPTION
**Pull request by pietrangelo**
profile for sonar analysis has been created to speedup the normal development test process.

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: admin-console/pom.xml [hunk 1]
```diff
@@ -98,21 +98,21 @@
...
-                    <testFailureIgnore>false</testFailureIgnore>
-                    <excludes>
-                        <exclude>**/*BaseTestCase*</exclude>
-                    </excludes>
-                    <includes>
-                        <include>**/AllTests.java</include>
-                    </includes>
-                </configuration>
-            </plugin>
+            <!--<plugin>-->
```
Comment (joewhite101) [inline admin-console/pom.xml]: Remove commented out code.

File: engine/pom.xml [hunk 2]
```diff
@@ -159,21 +159,21 @@
...
-                    <testFailureIgnore>false</testFailureIgnore>
-                    <excludes>
-                        <exclude>**/*BaseTestCase*</exclude>
-                    </excludes>
-                    <includes>
-                        <include>**/AllTests.java</include>
-                    </includes>
-                </configuration>
-            </plugin>
+            <!--<plugin>-->
```
Comment (joewhite101) [inline engine/pom.xml]: Remove

File: portal-ui/pom.xml [hunk 3]
```diff
@@ -97,21 +97,21 @@
...
-                    <testFailureIgnore>false</testFailureIgnore>
-                    <excludes>
-                        <exclude>**/*BaseTestCase*</exclude>
-                    </excludes>
-                    <includes>
-                        <include>**/AllTests.java</include>
-                    </includes>
-                </configuration>
-            </plugin>
+            <!--<plugin>-->
```
Comment (joewhite101) [inline portal-ui/pom.xml]: Remove

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**